DXRMapFixups/DeusEx/Classes/DXRFixupVandenberg.uc

```diff
@@ -4,6 +4,17 @@ var bool M14GaryNotDone;
 var bool M12GaryHostageBriefing;
 var int storedBotCount;// MJ12 bots in CMD
 
+function CheckConfig()
+{
+    local int i;
+
+    add_datacubes[i].map = "12_VANDENBERG_GAS";
+    add_datacubes[i].imageClass = class'Image12_Tiffany_HostagePic';
+    i++;
```

Comment:
should maybe have a defined location on the shelf near her, since these get spawned in PreFirstEntry their location gets randomized after being spawned, but without a location here then they just spawn on the floor somewhere which is a little lame

```suggestion
    add_datacubes[i].location = vect(x,y,z);
    i++;
```